src/main/java/com/amazonaws/services/kinesis/leases/impl/LeaseCoordinator.java

```diff
@@ -186,44 +185,46 @@ public LeaseCoordinator(ILeaseManager<T> leaseManager,
                 maxLeasesForWorker, maxLeasesToStealAtOneTime, maxLeaseRenewerThreadCount, metricsFactory);
     }
 
-    /**
-     * Constructor.
-     *
-     * @param leaseManager LeaseManager instance to use
-     * @param leaseSelector LeaseSelector instance to use
-     * @param workerIdentifier Identifies the worker (e.g. useful to track lease ownership)
-     * @param leaseDurationMillis Duration of a lease
-     * @param epsilonMillis Allow for some variance when calculating lease expirations
-     * @param maxLeasesForWorker Max leases this Worker can handle at a time
-     * @param maxLeasesToStealAtOneTime Steal up to these many leases at a time (for load balancing)
-     * @param metricsFactory Used to publish metrics about lease operations
-     */
     public LeaseCoordinator(ILeaseManager<T> leaseManager,
```

Comment:
No need for this to be `public`. Could you also add javadocs.

Reply:
Removed public. Added Javadocs.